extensions/amp-accordion/amp-accordion.md

```diff
@@ -0,0 +1,101 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-accordion"></a> `amp-accordion`
```

Comment:
Heads up that we're adding a new documentation structure on these pages. It might be nice to start using that here.

Check out https://github.com/ampproject/amphtml/pull/1841/files#diff-b06bcda88a7e36daa21769e39db62da6R17 for an example -- lines 17 to 36 with the header and the <table> are the key part.